_data/banking.yml

```diff
@@ -260,6 +260,14 @@ websites:
       software: Yes
       doc: https://www.dnb.no/privat/kundeservice/innlogging-nettbank.html
 
+    - name: Dutch Bangla Bank Limited (DBBL)
```

Comment:
If the acronym is commonly used then I would say keep it. It would make it easier to find the site via searching.